pkg/build/build.go

```diff
@@ -70,6 +70,8 @@ func getBuilder(targetOS, targetArch, vmType string) (builder, error) {
 		return fuchsia{}, nil
 	case targetOS == "akaros" && targetArch == "amd64" && vmType == "qemu":
 		return akaros{}, nil
+	case targetOS == "openbsd" && targetArch == "amd64" && vmType == "gce":
```

Comment:
Maybe we should start with "vmm" first? Because you already know how to build everything required for it.
The kernel object build will be reused for both, but image building will probably differ.

Reply:
Sounds like a plan.